README.md

```diff
@@ -27,6 +27,18 @@ Run `./gradlew fmt` to format your files. This also runs when you call `./gradle
 Run `./gradlew checkFmt` to verify your files are formatted. This also runs when you call `./gradlew check`.
 
 
+Configuration
+-----
+
+By default google-java-format version 1.3 is used. To change that you can explicitly add it to your dependencies.
+
+```groovy
+dependencies {
```

Comment:
No. Only dependencies added to the compile and apk configurations (plus their variant versions) are bundled into the apk. Since we are creating our own config it won't get packaged or even noticed by any other plugin.